packages/site/snapshots/latest/mosaic/author/aliases.mdx

```diff
@@ -65,47 +65,55 @@ sidebarData:
     data:
       level: 2
       link: /mosaic/author/index
-    childNodes: []
-  - id: /mosaic/author/markdown-syntax
-    fullPath: /mosaic/author/markdown-syntax.mdx
-    name: Markdown Syntax
-    priority: 5
-    data:
-      level: 2
-      link: /mosaic/author/markdown-syntax
-    childNodes: []
-  - id: /mosaic/author/frontmatter
-    fullPath: /mosaic/author/frontmatter.mdx
-    name: Frontmatter
-    priority: 4
-    data:
-      level: 2
-      link: /mosaic/author/frontmatter
-    childNodes: []
-  - id: /mosaic/author/aliases
-    fullPath: /mosaic/author/aliases.mdx
-    name: Aliases
-    priority: 3
-    data:
-      level: 2
-      link: /mosaic/author/aliases
-    childNodes: []
-  - id: /mosaic/author/ui-components
-    fullPath: /mosaic/author/ui-components.mdx
-    name: UI Components
-    priority: 2
-    data:
-      level: 2
-      link: /mosaic/author/ui-components
-    childNodes: []
-  - id: /mosaic/author/page-templates
-    fullPath: /mosaic/author/page-templates.mdx
-    name: Page Templates
-    priority: 1
-    data:
-      level: 2
-      link: /mosaic/author/page-templates
-    childNodes: []
+    childNodes:
```

Comment:
would it be possible to incorporate the changes from main and then regen the snapshot? otherwise it's reverting the sidebar back to this overly nested version

maybe we should be generating snapshots as part of ci or something to save this hassle..

Reply:
Yeah - would be great to get this automated!
In the meantime, I'll rebase and regen 👍

Reply:
Better now, @lilyvc ?

Had a couple of issues getting this sorted: an issue with my Turbo cache meant that running `yarn gen:snapshot` always gave me an empty snapshot 😬 
Then GitHub wasn't picking up the rebased commits. (looks like it's all there now)

Reply:
hm the preview site doesn't seem to be working now 😅

Reply:
Looks okay to me? 🤷 
https://mosaic-farlcajxp-mosaic-dev-team.vercel.app/mosaic/index

Reply:
ohh I see it was the vercel bot an hour ago. I guess it looks good then